global.json

```diff
@@ -1125,7 +1125,7 @@
       "name": "API Mapper",
       "priority": 0.883,
       "id": "rest-client",
-      "docs": "https://raw.githubusercontent.com/feedhenry-templates/fh-api-mapper/master/README.md",
+      "docs": "https://raw.githubusercontent.com/odra/fh-api-mapper/RHMAP-6206-fix/README.md",
```

Comment:
Why does the URL need to be changed for testing?  Ideally that PR will be tested and merged in isolation and then this one would be tested against master.